### ISSUE TITLE
Accessibility: incorrectly labeled buttons

### ISSUE THREAD

**Issue by shortwavesurfer2009** [PrimalHQ/primal-android-app#76]
## Describe the bug
The buttons below either say "button", "icon", "icon1", or are not read at all when using TalkBack

### Steps to reproduce
Steps to reproduce the behavior.
Open system settings > accessibility > talkback and turn it on
Open Primal
Tap the buttons highlighted with one finger

### Expected behavior
The buttons should be correctly labeled with their function

### Environment
 - Details that you think may affect.
Android 13
Primal 0.91.2 (Fdroid via IzzyOnDroid)

### Additional context
![Screenshot_20240214-081452_Primal](https://github.com/PrimalHQ/primal-android-app/assets/116814522/09c998a3-ce11-4ba2-a0d3-a62634f6b5e6)
![Screenshot_20240214-081622_Primal](https://github.com/PrimalHQ/primal-android-app/assets/116814522/6a5cc016-0698-447b-885f-07e65e4527ab)
![Screenshot_20240214-081635_Primal](https://github.com/PrimalHQ/primal-android-app/assets/116814522/c8798259-af14-457d-a465-3ac969aa2136)
![Screenshot_20240214-081639_Primal](https://github.com/PrimalHQ/primal-android-app/assets/116814522/15f9e62e-993e-4571-9dd4-9a09d0d02e98)
![Screenshot_20240214-081643_Primal](https://github.com/PrimalHQ/primal-android-app/assets/116814522/ff5c5358-0ff8-48ab-aac3-7d5dc6a5bfa4)
![Screenshot_20240214-081649_Primal](https://github.com/PrimalHQ/primal-android-app/assets/116814522/0d435278-f210-4c39-871d-cd09cb48e3d8)
![Screenshot_20240214-081658_Primal](https://github.com/PrimalHQ/primal-android-app/assets/116814522/1246e847-f0fd-4784-9d28-070dc5f59304)
![Screenshot_20240214-081710_Primal](https://github.com/PrimalHQ/primal-android-app/assets/116814522/7c525ed0-c2e1-4912-a85a-ff5d5028175a)
![Screenshot_20240214-081713_Primal](https://github.com/PrimalHQ/primal-android-app/assets/116814522/4814d06b-290c-499c-a9f1-0a9cc9f3fab6)
![Screenshot_20240214-081717_Primal](https://github.com/PrimalHQ/primal-android-app/assets/116814522/94b7a2ce-356a-40c2-ade7-f2e228d7819e)
![Screenshot_20240214-081718_Primal](https://github.com/PrimalHQ/primal-android-app/assets/116814522/cb3e1164-dac6-4bd5-9c10-f19ccf23f39e)
![Screenshot_20240214-081724_Primal](https://github.com/PrimalHQ/primal-android-app/assets/116814522/c7800d8a-25cb-409e-8f95-bb1914bc7482)
![Screenshot_20240214-081802_Primal](https://github.com/PrimalHQ/primal-android-app/assets/116814522/11ca9662-f690-4d04-bf40-97bb68da51e4)
![Screenshot_20240214-081826_Primal](https://github.com/PrimalHQ/primal-android-app/assets/116814522/ddcd3461-6f0d-4b18-99b3-77cc0ebe5610)
![Screenshot_20240214-081831_Primal](https://github.com/PrimalHQ/primal-android-app/assets/116814522/e1bf35fc-16c4-422d-a338-2089d9c98a7f)
![Screenshot_20240214-081835_Primal](https://github.com/PrimalHQ/primal-android-app/assets/116814522/8d875d4b-d4fe-4dd0-af75-8574006e1303)
![Screenshot_20240214-081756_Primal](https://github.com/PrimalHQ/primal-android-app/assets/116814522/471bc5c6-0d06-44c6-9fab-9aefb7f69bdc)
![Screenshot_20240214-081840_Primal](https://github.com/PrimalHQ/primal-android-app/assets/116814522/5d5e4900-1dc8-46f9-b572-e9e4414a5fae)
![Screenshot_20240214-081851_Primal](https://github.com/PrimalHQ/primal-android-app/assets/116814522/4e864bb8-bb4a-444f-b5fd-0895cedfcd9e)
![Screenshot_20240214-081903_Primal](https://github.com/PrimalHQ/primal-android-app/assets/116814522/e9fdf1b2-2610-4326-8367-0f949090c0a8)
![Screenshot_20240214-081923_Primal](https://github.com/PrimalHQ/primal-android-app/assets/116814522/eae87d60-0bd3-4285-840a-7284ab9c499c)
![Screenshot_20240214-082031_Primal](https://github.com/PrimalHQ/primal-android-app/assets/116814522/e9165d6d-a35d-4658-a61b-94a1361e8ca8)
![Screenshot_20240214-082044_Primal](https://github.com/PrimalHQ/primal-android-app/assets/116814522/e703a207-0578-4df5-b8fe-fe5f4bd5519f)
![Screenshot_20240214-082130_Primal](https://github.com/PrimalHQ/primal-android-app/assets/116814522/1afc4ff7-f3c5-4dfb-8bea-4d3a08f3411f)
![Screenshot_20240214-082156_Primal](https://github.com/PrimalHQ/primal-android-app/assets/116814522/0f8c0ec1-2704-499b-9927-ce9129955b9f)
![Screenshot_20240214-082304_Primal](https://github.com/PrimalHQ/primal-android-app/assets/116814522/5be74c96-ce74-4f2a-b918-67c601a70fb6)
![Screenshot_20240214-082309_Primal](https://github.com/PrimalHQ/primal-android-app/assets/116814522/a446b4e5-3783-4ae8-9dd7-8b783e7dabe3)
![Screenshot_20240214-082312_Primal](https://github.com/PrimalHQ/primal-android-app/assets/116814522/10117375-67d4-43cb-b231-c15c8e101dda)
![Screenshot_20240214-082345_Primal](https://github.com/PrimalHQ/primal-android-app/assets/116814522/48115948-aac8-46f2-9d0e-41a637975b86)
![Screenshot_20240214-082355_Primal](https://github.com/PrimalHQ/primal-android-app/assets/116814522/09160920-1741-4fa1-9cd4-e49f68aa11e7)

**Comment by shortwavesurfer2009** [comment]
I did not test the login screens as I am currently logged in already, but everything else looks perfect. So I assume the login screen ones got fixed as well. Thanks for the great work.
